### PR TITLE
Fix Part 2 and Part 3 encoding. Convert UTF-8-BOM to UTF-8

### LINKED ISSUE #31: Having problem opening "Part 2 Forward Propagation.ipynb", "Part 3 Gradient Descent.ipynb" and "Part 4 Backpropagation.ipynb" via Jupyter notebook
Hello those files won't open via jupyter notebook
image

Windows 10 1809 home edition. Anaconda 4.7.12
image
Thanks in advance

### PR DESCRIPTION
**Pull request by const-bon**
## what
* Fix https://github.com/stephencwelch/Neural-Networks-Demystified/issues/31
* Fix Part 2, Part 3 and Part 4 encoding. Convert UTF-8-BOM to UTF-8
* Fix incorrect doublequotes escape in Part 4

## Why
* https://github.com/stephencwelch/Neural-Networks-Demystified/issues/31
* Can't open Part 2, Part 3, Part 4

### CODE REVIEW

**Comment by const-bon** [comment]
TODO: Fix Part 4

**Comment by const-bon** [comment]
Part 4 fixed.
